lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderServiceImpl.java

```diff
@@ -139,4 +188,25 @@ public void reportProviderEnabled(String provider) {
   public Map<PendingIntent, LocationRequest> getPendingIntents() {
     return intentToRequest;
   }
+
+  private LocationAvailability createLocationAvailability() {
```

Comment:
I wonder if this method should take the check one step further and verify not only at least one location provider is enabled but also that `LocationManager#getLastKnownLocation(provider)` returns a non-null value for at least one provider?